src/box/session.h

```diff
@@ -172,6 +172,20 @@ struct session_vtab {
 	 */
 	int64_t
 	(*sync)(struct session *session);
+
+	/**
+	 * Only supported by IPROTO sessions.
```

Comment:
Why add it to the vtab then? You can call the iproto function directly.

Reply:
Fixed: removed this function from the session virtual table.